src/admin/admin.resolver.ts

```diff
@@ -1,19 +1,20 @@
-import { Args, Int, Query, Resolver } from '@nestjs/graphql';
+import { UseGuards, UseInterceptors } from '@nestjs/common';
+import { Args, Context, Mutation, Resolver } from '@nestjs/graphql';
 
-import { AdminService } from './admin.service';
+import { LoginInput } from './dto';
 import { Admin } from './entities/admin.entity';
+import { LocalGqlAdminGuard } from './guards';
+import { SetCookieInterceptor } from './interceptors';
 
 @Resolver(() => Admin)
 export class AdminResolver {
-  constructor(private readonly adminService: AdminService) {}
-
-  @Query(() => [Admin], { name: 'admins' })
-  findAll() {
-    return this.adminService.findAll();
-  }
-
-  @Query(() => Admin, { name: 'admin' })
-  findOne(@Args('id', { type: () => Int }) id: number) {
-    return this.adminService.findOne(id);
+  @Mutation(() => Admin, { name: 'login', nullable: true })
+  @UseGuards(LocalGqlAdminGuard)
+  @UseInterceptors(
```

Comment:
where is `httpOnly` cookie for admin information?

Reply:
this cookie is just for FE guys and should not be httpOnly, anyone can access to it